# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/vr/service/VRService.h
# Commit: 75b04de7e99c
# Full Hash: 75b04de7e99cab7bed5cc8b0f2f6d52ddf174bf3
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-25 09:39:13
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r 91c3acdb2454 -r 75b04de7e99c gfx/vr/service/VRService.h
--- a/gfx/vr/service/VRService.h	Fri May 24 11:32:20 2019 +0000
+++ b/gfx/vr/service/VRService.h	Fri May 24 11:32:54 2019 +0000
@@ -66,10 +66,7 @@
 #if defined(XP_WIN)
   HANDLE mMutex;
 #endif
-  // We store the value of gfxPrefs::VRProcessEnabled() in mVRProcessEnabled.
-  // This allows us to read the value in the VRService destructor, after
-  // gfxPrefs has been shut down.  We should investigate why gfxPrefs
-  // is shutting down earlier - See bug xxx
+  // We store the value of StaticPrefs::VRProcessEnabled() in mVRProcessEnabled.
   bool mVRProcessEnabled;
 
   bool IsInServiceThread();